docs/modules/mysql.md

```diff
@@ -160,4 +290,52 @@ Drops the applicable user from the MySQL DBMS.
 
 ##### Deprovision
 
-Deletes the database from the MySQL DBMS. The DBMS itself is not deprovisioned.
\ No newline at end of file
+Deletes the database from the MySQL DBMS. The DBMS itself is not deprovisioned.
+
+##### Examples
+
+###### Kubernetes
+
+The `contrib/k8s/examples/mysql/mysql-database-only-instance.yaml` can be used to provision the `database-only` plan. This can be done with the following example:
+
+```console
+kubectl create -f contrib/k8s/examples/mysql/mysql-database-only-instance.yaml
+```
+
+You can then create a binding with the following command:
+
+```console
+kubectl create -f contrib/k8s/examples/mysql/mysql-database-only-binding.yaml
+```
+
+###### Cloud Foundry
+
+Using the `cf` cli, you can provision the `database-only` plan of this service with the following command:
+
+```console
+cf create-service azure-mysql-database-only database-only mysql-database-only -c '{
+    "parentAlias" : "679aab6d-39e7-4a41-8b45-49975569079c"
+}
+'
+```
+
+Note: this uses the alias provided in the DBMS only example
```

Comment:
DBMS-only

Also, the note above was terminated with a period, but here is not. Suggest standardizing.